src/components/button/button.js

```diff
@@ -1,10 +1,30 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-export const Button = ({ children }) => {
-  return <button className="button is-danger is-outlined">{children}</button>;
+export const Button = ({ children, onClick, type, inverted }) => {
+  const additionalClasses = [];
+
+  if (type) {
+    additionalClasses.push(`is-${type}`);
+  }
+
+  if (inverted) {
+    additionalClasses.push("is-outlined");
+  }
+
+  return (
+    <button
+      onClick={onClick}
+      className={"button is-primary " + additionalClasses.join(" ")}
+    >
+      {children}
+    </button>
+  );
 };
 
 Button.propTypes = {
-  children: PropTypes.node.isRequired
+  children: PropTypes.node.isRequired,
+  onClick: PropTypes.func.isRequired,
+  type: PropTypes.oneOf(["primary", "danger"]),
```

Comment:
We could set `type: primary` as default. Please check [this](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) article, same for `inverted`